pkg/kubectl/describe.go

```diff
@@ -830,7 +830,7 @@ func (d *PersistentVolumeDescriber) Describe(namespace, name string, describerSe
 		w := &PrefixWriter{out}
 		w.Write(LEVEL_0, "Name:\t%s\n", pv.Name)
 		printLabelsMultiline(w, "Labels", pv.Labels)
-		w.Write(LEVEL_0, "StorageClass:\t%s\n", storageutil.GetStorageClassAnnotation(pv.ObjectMeta))
```

Comment:
I don't think it's necessary, since this would be an issue anytime someone adds a new field.  The main requirement is that older clients can create and view their older objects, which this handles.